nservicebus/licensing/license-management.md

```diff
@@ -13,6 +13,8 @@ There are several ways to make sure that your NServiceBus endpoints pick up and
 
 ## Using the Registry
 
+Using the registry to store your license information is preferred as all platform tools can access this information easily. This includes NServiceBus endpoints, ServiceControl, ServiceInsight, and ServiceMatrix. (ServicePulse determines licensing status by querying the ServiceControl API.) Using the registry ensures that all the platform tools can access the license status without requiring additional complexity on every deployment.
```

Comment:
Since SC, SM and Endpoints are most likely on different machines adding the registry to the mix actually makes it more complex. The least deployment steps for enspoints is to ship the license on the filesystem